sbnanaobj/StandardRecord/SRTrueInteraction.h

```diff
@@ -45,10 +43,16 @@ namespace caf
     float      eccqe;             //!< CCQE energy
     float      baseline;          //!< Distance from decay to interaction
 
+    int        parent_dcy_mode;   //!< Parent hadron/muon decay mode
```

Comment:
I don't know how strict we want to be on requiring everything to have a default value in the constructor?

Reply:
We definitely want to be really strict about not ever allowing uninitialized values into the final CAFs. Those are a complete pain. Requiring default values in the constructor does sound like a sensible step towards that goal.

Reply:
Fixed.

Although there are more than a couple other structures that do not default initialize. If this is a concern it's probably worth doing an audit.